theseus_gui/src/components/ui/Breadcrumbs.vue

```diff
@@ -1,5 +1,14 @@
 <template>
   <div class="breadcrumbs">
+    <Button
+      v-if="breadcrumbs.length > 1"
+      class="breadcrumbs__back transparent"
+      color="primary"
+      icon-only
+      @click="$router.back()"
+    >
+      <LeftArrowIcon />
+    </Button>
```

Comment:
I think this should always show and just behave like the browser back button. We could do a forward button too and just have it styled as a normal button